webapp/src/components/properties/lastModifiedBy/lastModifiedBy.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useContext} from 'react'
+
+import {WorkspaceUsersContext, WorkspaceUsers} from '../../../user'
+import {Card} from '../../../blocks/card'
+import {BoardTree} from '../../../viewModel/boardTree'
+
+type Props = {
+    card: Card,
+    boardTree?: BoardTree,
+}
+
+const LastModifiedBy = (props: Props): JSX.Element => {
+    let latestBlock = props.boardTree?.allBlocks.filter((block) => block.parentId === props.card.id || block.id === props.card.id).sort((a, b) => b.updateAt - a.updateAt)[0]
+    latestBlock = latestBlock || props.card
```

Comment:
Same as above. Could we refactor into if statement with each function call being on a new line to avoid having one long line. Thanks

Reply:
Done.